app/assets/javascripts/discourse/app/templates/modal/discard-draft.hbs

```diff
@@ -6,8 +6,5 @@
 
 <div class="modal-footer">
   <DButton @icon="far-trash-alt" @label="post.cancel_composer.discard" @class="btn-danger discard-draft" @action={{action "destroyDraft"}} />
-  {{#if this.model.canSaveDraft}}
```

Comment:
IMO it's fine to still keep this in the modal, just gives another avenue to make sure your draft is saved

Reply:
In which case `saveDraftAndClose` can stick around in `discard-draft.js`

Reply:
Also it seems like if you keep this all of the current tests will still pass